somnio_starter/__brick__/{{project_name.snakeCase()}}/lib/authentication/delete_account/bloc/delete_account_state.dart

```diff
@@ -19,10 +19,16 @@ enum DeleteAccountStatus {
 
 class DeleteAccountState extends Equatable {
   const DeleteAccountState({
-    this.status = DeleteAccountStatus.initial,
-    this.email = const Email.pure(),
-    this.password = const Password.pure(),
+    required this.status,
+    required this.email,
+    required this.password,
   });
+
+  const DeleteAccountState.initial() : this(
```

Comment:
Is this necessary? Let's follow the same approach we did for other features

Reply:
I saw we are doing the same approach in the login feature